exercises/resistor-color/resistor_color_test.exs

```diff
@@ -0,0 +1,37 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("resistor_color.exs", __DIR__)
+end
+
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
+
+defmodule ResistorColorTest do
+  use ExUnit.Case
+
+  @colors ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
```

Comment:
move this to a local variable inside of the `returns all colors` test

Reply:
@neenjaw done!